centralbank.py

```diff
@@ -13,6 +14,26 @@ def __init__(self):
         self.twelvemonth_CPI = 0
         self.feedback_counter = 0
         self.prices_list = []
+        self.economy_money = money_supply
+        self.warnings = {}
+        self.aid_budget = 1000000
```

Comment:
Perhaps this should be a parameter in `isleconfig.py` since its impact depends on the relation to other values fixed there.